cli/account/cli.py

```diff
@@ -54,7 +51,10 @@ def get_balance(address, node_url):
         'node_address': str(node_url) + ':8080',
     })
 
-    account_service = Account(service=remme)
-    balance = loop.run_until_complete(account_service.get_balance(address=address))
+    result, errors = Account(service=remme).get_balance(address=address)
+
+    if errors is not None:
```

Comment:
Why it's needed here, if `Account(service=remme).get_balance(address=address)` always return `error=None`.

Reply:
@Alladin9393, I think, In the future, we will have general exception cathing in every service. Could we leave it here although there is no errors.

Reply:
@dmytrostriletskyi, ok.